external/jshint/jshint.js

```diff
@@ -58488,7 +58488,9 @@ var warnings = {
 	W101: i18n._("Line is too long."),
 	W102: i18n._("Trailing whitespace."),
 	W103: i18n._("The '{a}' property is deprecated."),
-	W104: i18n._("'{a}' is only available in JavaScript 1.7."),
+	W104: i18n._("It looks like you're trying to use '{a}'. '{a}' is a " +
+        "JavaScript 1.7 feature and is not currently supported in this " +
```

Comment:
I'll edit this in a bit as I cannot currently. I said "JavaScript 1.7" instead of "ES6" as it matched the phrasing of the original message.